packages/react/src/FAQ/FAQ.tsx

```diff
@@ -104,14 +113,37 @@ function FAQSubheading({children, className, as = 'h3', ...rest}: FAQSubheadingP
   )
 }
 
+type FAQGroupProps = BaseProps<HTMLUListElement> & {
+  children: React.ReactNode | React.ReactNode[]
+  'aria-labelledby': string
+}
+
+function FAQGroup({className, children}: FAQGroupProps) {
+  return (
+    // eslint-disable-next-line jsx-a11y/no-redundant-roles
```

Comment:
> sev 2) For all examples, the accordion controls are not using list semantics. Screen reader users will not be made aware of the grouped nature of the controls, nor how many items are in the collection.

> Solution: Use an unordered list ul to wrap the related accordion controls, with an li wrapping each individual item. Add an aria-labelledby attribute to the ul, and reference the ID of the section heading. If list-style: none is applied with CSS, be sure to add role="list" to the ul element to ensure that all screen readers report the list structure. Some screen reader/browser combinations will remove list semantics if no bullets are present, and the explicit role declaration restores them.

This needed to be added to follow this guidance. Maybe the linter can be updated so we can remove this in the future.